app/src/main/java/com/globant/example/mentorapp/di/Module/ApplicationModule.java

```diff
@@ -0,0 +1,92 @@
+package com.globant.example.mentorapp.di.Module;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Singleton;
+
+import com.globant.example.mentorapp.MentorApplication;
+import com.globant.example.mentorapp.home.domain.interactor.data.remote.APIService;
+import com.globant.example.mentorapp.home.domain.interactor.data.remote.OkHttpConstants;
+import com.globant.example.mentorapp.home.domain.interactor.data.util.ApiUtils;
+import com.globant.example.mentorapp.home.domain.model.UserEntity;
+import com.squareup.otto.Bus;
+import com.squareup.otto.ThreadEnforcer;
+
+import android.arch.lifecycle.MutableLiveData;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.StaggeredGridLayoutManager;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Class to control UserList injections
+ * Created by oscar.amezquita on 8/06/2017.l
+ */
+@Module
+public class ApplicationModule {
+
+    public MentorApplication application;
+    public Bus bus;
+
+    public ApplicationModule(MentorApplication application) {
+
+        this.application = application;
+        bus = new Bus(ThreadEnforcer.ANY);
+    }
+
+    @Provides
+    protected StaggeredGridLayoutManager provideLinearLayoutManager() {
+        return new StaggeredGridLayoutManager(2, LinearLayoutManager.VERTICAL);
```

Comment:
This number represents the number of columns of the StaggeredGridLayoutManager.  Passed to Integer constants.